scss/themes/_package.scss

```diff
@@ -0,0 +1,90 @@
+@mixin oTopperAnchorLink {
+	@include oTypographySans(0, $progressive: true);
+	@include oTypographyMargin($top: 8);
+	display: inline-block;
+	width: 110px;
+	padding: 5px 0 0 10px;
+	border-top: 1px solid oColorsGetColorFor('link', 'text');
+	border-left: 1px solid oColorsGetColorFor('link', 'text');
+	border-bottom-width: 0;
+	white-space: nowrap;
+	overflow: visible;
+
+	@include oGridRespondTo(S) {
+		width: 140px;
+	}
+
+	&:after {
+		@include oIconsGetIcon('arrow-down', oColorsGetColorFor('link', 'text'), 20, $iconset-version: 1);
+		content: '';
+		display: block;
+		position: relative;
+		top: 4px;
+		left: -4px;
+		margin-top: -2px;
+	}
+}
+
+@mixin oTopperThemePackage {
+	@include oTypographyMargin($bottom: 10);
+
+	%_o-topper__tag,
+	.o-topper__headline {
+		@include oTypographySans(6, $line-height: 1em);
+		font-weight: 400;
+
+		@include oGridRespondTo(L) {
+			@include _oTypographyProgressiveFontFallbackSize('sans', 8);
+		}
+	}
+
+	.o-topper__headline {
+		&:after {
+			border-bottom-width: 1px;
+		}
+	}
+}
+
+@mixin oTopperPackageSequenceId {
```

Comment:
`oTopperPackageSequenceId` is no longer used?

Reply:
next is outputting that element sometimes, i'm trying to find out when

Reply:
we can remove this 👌